quality-gate-manifest.json

```diff
@@ -0,0 +1,124 @@
+{
+  "$schema": "https://raw.githubusercontent.com/govuk-one-login/quality-gates/refs/tags/v0.1.0/schemas/schema.json",
+  "services": [
+    {
+      "service-tag": "id-check",
```

Comment:
Hi! I'm working on similar changes for the ID Check backends.
Out of curiousity how did you decide to use this `service-tag` value?
Has it been used elsewhere in code metrics?

Reply:
I really don't know how to choose a service tag - 

I added this value based on [this conversation](https://gds.slack.com/archives/C090BJ79JU9/p1763569430937149?thread_ts=1763568327.793909&cid=C090BJ79JU9). I gathered that the `service-tag` value can be changed as we need so I was counting on seeing the output in code metrics and updating later.

Do you know if we can we see service tags in code metrics?

For the format, I'm using `kebab-case` to match the examples:
- https://github.com/govuk-one-login/authentication-api/pull/7000/files
- https://github.com/govuk-one-login/ipv-spot/pull/1673/files